### PR TITLE
REL-3272: Add GMOS OVI and OVIC filters to the ITC

### PR DESCRIPTION
**Pull request by andrewwstephens**
This PR adds the the new GMOS OVI filters to the ITC:
- Add transmission profiles
- Add filters to ITC web forms

I rearranged the ITC web form filter lists slightly to be a little more consistent between GMOS-N and GMOS-S.

I also removed the barcode from the filter ID to make them more like the other filters, as this sets the name of the filter transmission files.

### CODE REVIEW

File: bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java [hunk 1]
```diff
@@ -171,8 +171,8 @@ public DisperserNorth parse(String name, DisperserNorth defaultValue) {
         OIIIC_G0319("OIIIC_G0319", "OIIIC", "0.514"),
         HeII_G0320("HeII_G0320", "HeII", "0.468"),
         HeIIC_G0321("HeIIC_G0321", "HeIIC", "0.478"),
-        OVI_G0345_20000053("OVI_G0345", "OVI", "0.6835"),
-        OVIC_G0346_20000054("OVIC_G0346", "OVIC", "0.678"),
+        OVI_G0345("OVI_G0345", "OVI", "0.6835"),
```
Comment (cquiroz) [inline bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java]: Could this cause troubles with existing programs? Do we have these filters on the current production ODB?

Comment (sraaphorst) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java]: I recently added them to the 2018A OCS and they are new, so I am thinking this should be fine.

Comment (cquiroz) [reply bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java]: 👍

**Comment by arturog8m** [comment]
This is a test - please ignore (testing JIRA integration)

**Pull request merged**